packages/react/test/specs/lib/felaRenderer-test.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { createSnapshot } from 'jest-react-fela'
+import { EmptyThemeProvider } from 'test/utils'
+import Box from 'src/components/Box/Box'
+import Animation from 'src/components/Animation/Animation'
+import Icon from 'src/components/Icon/Icon'
+import Provider from 'src/components/Provider/Provider'
+import Text from 'src/components/Text/Text'
+import { felaRenderer } from 'src/lib'
+import { teams } from 'src/themes'
+
+test('css fallback value', () => {
+  const snapshot = createSnapshot(
+    <EmptyThemeProvider>
+      <Box styles={{ color: 'red' }} />
+    </EmptyThemeProvider>,
+    {},
+    felaRenderer,
+  )
+  expect(snapshot).toMatchSnapshot()
+})
+
+test('keyframe', () => {
+  const spinner = {
+    keyframe: ({ fromColor, toColor }) => ({
+      from: {
+        color: fromColor,
+      },
+      to: {
+        color: toColor,
+      },
+    }),
+    keyframeParams: {
+      fromColor: 'red',
+      toColor: 'blue',
+    },
+    duration: '5s',
+  }
+
+  const snapshot = createSnapshot(
+    <Provider theme={teams}>
+      <Provider theme={{ animations: { spinner } }}>
+        <Animation name="spinner">
+          <Icon name="umbrella" circular bordered />
+        </Animation>
+      </Provider>
+    </Provider>,
+    {},
+    felaRenderer,
+  )
+  expect(snapshot).toMatchSnapshot()
+})
+
+test('keyframe returning css fallback value', () => {
+  const steps = ['0%', '100%']
+
+  const spinner = {
+    keyframe: ({ steps }) => {
+      const obj = {}
+      steps.forEach((step: string, index) => {
+        ;(obj as any)[step] = { opacity: 0.28 }
```

Comment:
This (the `opacity`) would have to be an array to test 'array returned by keyframe results in CSS fallback values'